youtube_dl/extractor/podchaser.py

```diff
@@ -0,0 +1,98 @@
+# coding: utf-8
+from __future__ import unicode_literals
+import re
+from datetime import datetime
+from ..utils import float_or_none, try_get, str_to_int
+from ..compat import compat_str
+from .common import InfoExtractor
+
+
+class PodchaserIE(InfoExtractor):
+    _VALID_URL = r'''(?x)
+        https?://(?:www\.)?podchaser\.com/
+        (?:(?:podcasts)|(?:creators))
+        /[\w\-]+-
```

Comment:
To put `-` in `[]`, just make it the last item, hence `[\w-]`

Reply:
- [x] see 89ef8b9b99a5cdf24a81dc3070106302bef64724